devsetup/scripts/standalone.sh

```diff
@@ -69,19 +69,8 @@ source ${SCRIPTPATH}/common.sh
 # export NTP_SERVER=pool.ntp.org
 
 if [[ ! -f $REPO_SETUP_CMDS ]]; then
-    cat <<EOF > $REPO_SETUP_CMDS
-set -ex
-sudo dnf remove -y epel-release
-sudo dnf update -y
-sudo dnf install -y vim git curl util-linux lvm2 tmux wget
-URL=https://trunk.rdoproject.org/centos9-wallaby/component/tripleo/current-tripleo/
-RPM_NAME=\$(curl \$URL | grep python3-tripleo-repos | sed -e 's/<[^>]*>//g' | awk 'BEGIN { FS = ".rpm" } ; { print \$1 }')
-RPM=\$RPM_NAME.rpm
-sudo dnf install -y \$URL\$RPM
-sudo -E tripleo-repos -b wallaby current-tripleo-dev ceph --stream
-sudo dnf repolist
-sudo dnf update -y
```

Comment:
This line should be moved to somewhere after downstream repos are enabled to have updated base opearting system

Reply:
For standalone jobs the repo_cmd is coming from /home/zuul/cdn_subscription_repos.sh. Not sure, if the update command is there. This could be examined separately  from this PR.